src/battle_anim.c

```diff
@@ -479,6 +484,126 @@ static void Cmd_createsprite(void)
     gAnimVisualTaskCount++;
 }
 
+static void CreateSpriteOnTargets(const struct SpriteTemplate *template, u8 argVar, u8 battlerArgIndex, u8 argsCount, bool32 overwriteAnimTgt)
+{
+    u32 i;
+    u8 targets[MAX_BATTLERS_COUNT];
+    int ntargets;
+    s16 subpriority;
+    
+    for (i = 0; i < argsCount; i++)
+    {
+        gBattleAnimArgs[i] = T1_READ_16(sBattleAnimScriptPtr);
+        sBattleAnimScriptPtr += 2;
+    }
+
+    if (argVar & 0x80)
```

Comment:
Since getting subpriority is the same here as in the Cmd_createsprite function, I think it's worth seperating these two duplicates of code into a function like 'GetSubpriorityForMoveAnim'.